metecho/api/models.py

```diff
@@ -50,35 +50,35 @@
 
 
 class OrgType(models.TextChoices):
-    PRODUCTION = ("Production", "Production")
-    SCRATCH = ("Scratch", "Scratch")
-    SANDBOX = ("Sandbox", "Sandbox")
-    DEVELOPER = ("Developer", "Developer")
+    PRODUCTION = "Production"
+    SCRATCH = "Scratch"
+    SANDBOX = "Sandbox"
+    DEVELOPER = "Developer"
 
 
 class ScratchOrgType(models.TextChoices):
-    DEV = ("Dev", "Dev")
+    DEV = "Dev"
     QA = ("QA", "QA")
```

Comment:
Why not this one?

Reply:
So the syntax is:

```
<python name> = (<DB value>, <admin label>)
```

When you leave out `admin label` Django generates it by "title-casing" `python name`. In this case `QA` turns into `Qa`, which looks weird so we manually set it to `QA`.

Reply:
Ah, and that explains the migration updates as well 👍